src/vs/workbench/contrib/welcomeGettingStarted/browser/featuredExtensionService.ts

```diff
@@ -86,12 +86,12 @@ export class FeaturedExtensionsService extends Disposable implements IFeaturedEx
 
 		const extensions = await Promise.race([
 			this.tasExperimentService?.getTreatment<string>('welcome.featured.item'),
-			new Promise<string | undefined>(resolve => setTimeout(() => resolve(''), 2000))
+			new Promise<string | undefined>(resolve => setTimeout(() => resolve(''), 100))
 		]);
```

Comment:
FYI there's the `raceTimeout` helper for this:

https://github.com/microsoft/vscode/blob/8ffaf6b7cc3306b36627b1d6681ae2a281c98dd1/src/vs/base/common/async.ts#L114-L125